### PR TITLE
Add react-router

### PR DESCRIPTION
**Pull request by emsi-iggy**

### CODE REVIEW

**Review by tomdziwood — APPROVED** [review]

**Review by tomdziwood — CHANGES_REQUESTED** [review]
File "package.json" is weird.

File: src/main/frontend/src/App/App.js [hunk 1]
```diff
@@ -525,6 +535,23 @@ class App extends Component {
         return true;
     };
 
+    checkIfProjectInURLExists = () => {
+        const { projects } = this.state;
+        const url = window.location.href.toString();
+        const urlSplitted = url.split('/');
```
Comment (dominieq) [inline src/main/frontend/src/App/App.js:541]: Add empty line between variable declarations and logic.

**Review by dominieq — CHANGES_REQUESTED** [review]

File: src/main/frontend/src/App/App.js (hunk 1 above)
Comment (emsi-iggy) [reply src/main/frontend/src/App/App.js:541]: [solved in this commit](https://github.com/dominieq/rule-studio/pull/82/commits/d35c1812970d19212880a0f672fe77145fa75d76)

**Review by dominieq — APPROVED** [review]

**Review by tomdziwood — APPROVED** [review]

**Comment by dominieq** [comment]
Fixes #83

**Pull request merged**